### DISCUSSION TITLE
Overleaf 打开 paper-translation 有`_ must be inside math mode`虚警

### DISCUSSION THREAD

**Discussion by YDX-2147483647** [BITNP/BIThesis#402] [General]
https://github.com/overleaf/overleaf/issues/1185

---

<small>
@CZLeader 告诉我的。
</small>

**Comment by YDX-2147483647** [comment]
> Hi there, the syntax parser unfortunately will never be perfect, but the trick from [overleaf.com/learn/how-to/Code_Check](https://www.overleaf.com/learn/how-to/Code_Check) shall work:
> 
> ```latex
> %%begin novalidate
> \documentclass[under_score]{foo}
> %%end novalidate
> ```
> 
> Note that we in general do not provide support for the Overleaf Community Edition. For issues that are exhibiting themselves also on the SaaS version at [www.overleaf.com](http://www.overleaf.com), you can contact us at [support@overleaf.com](mailto:support@overleaf.com). I hope this helps.

https://github.com/overleaf/overleaf/issues/1185#issuecomment-2009336096

**Comment by fky2015** [comment]
那可以一个 PR 修一下

**Reply by YDX-2147483647** [reply]
我认为没必要。一是不影响功能，没有实际效果；二是Overleaf还有别的虚警（bithesis、headerImage、xiheiFont等都被当成拼写错误），适配也适配不全。

**Reply by fky2015** [reply]
OK，那我们先忽略这个问题。

**Comment by YDX-2147483647** [comment]
补充：`\documentclass[…, ctex={fontset=windows}]{bithesis}`也会报假错。

```log
invalid documentclass option: unexpected close group }
```

**Comment by YDX-2147483647** [comment]
为方便大家查看，这个问题归档到了 [🥑 疑难杂症 (bithesis.bitnp.net/faq)](https://bithesis.bitnp.net/faq/)，后续将在那边更新。如果遇到新问题，欢迎继续在 GitHub Discussions 讨论。

https://bithesis.bitnp.net/faq/overleaf-false-error.html
https://github.com/BITNP/BIThesis-wiki/blob/ecf97efcb3f6e2a46d7b937d8bccfbf1f14eba2d/wiki/faq/overleaf-false-error.md
